.ci/magic-modules/point-to-submodules.sh

```diff
@@ -44,11 +44,20 @@ for PRD in "${PRODUCT_ARRAY[@]}"; do
 done
 
 if [ "$TERRAFORM_ENABLED" = "true" ]; then
-  git config -f .gitmodules submodule.build/terraform.branch "$BRANCH"
-  git config -f .gitmodules submodule.build/terraform.url "git@github.com:$GH_USERNAME/terraform-provider-google.git"
-  git submodule sync build/terraform
-  ssh-agent bash -c "ssh-add ~/github_private_key; git submodule update --remote --init build/terraform"
-  git add build/terraform
+  for VERSION in '' 'beta'; do
```

Comment:
This is at least as complicated as just doubling up the code below - same comment as above re: duplication vs iteration.